src/types.ts

```diff
@@ -42,7 +42,7 @@ export type AggregatedRecord = {
   seroprevalence: number;
 };
 
-export type Filters = {
+export type FiltersConfig = {
```

Comment:
why the name change?

Reply:
We already had a `Filters` component, so a `Filters` type was just confusing and hard to work with